extensions/mongodb-client/deployment/src/main/java/io/quarkus/mongodb/deployment/MongoClientBuildTimeConfig.java

```diff
@@ -29,4 +29,10 @@
      */
     @ConfigItem(name = "force-default-clients")
     public boolean forceDefaultClients;
+
+    /**
+     * Configuration for DevServices. DevServices allows Quarkus to automatically start MongoDB in dev and test mode.
+     */
+    @ConfigItem
+    public DevServicesBuildTimeConfig devServices;
```

Comment:
For now I have not enabled dev services for the named connections - just the default one